src/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { FC } from "react"
+import { SidebarBox, ComponentBox, InfoBox } from "./styled"
+
+export const Sidebar: FC<SidebarProps> = ({ info, children, right = false }: SidebarProps) => (
+  <ComponentBox side={right ? "row-reverse" : "row"}>
```

Comment:
I suggest to pass the `right` prop into styled and pick the style there. "row-reverse" and other css syntax shouldn't appear on component level.

Reply:
@Azein done